shell/plugins/steve/worker/web-worker.basic.js

```diff
@@ -91,6 +95,12 @@ const workerActions = {
 
     // Delete the schema from the map, so if it comes back we don't ignore it if the hash is the same
     delete state.schemas[id];
+  },
+  watch: (msg) => {
+    redispatch({ send: msg });
+  },
+  createWatcher: (msg) => {
+    redispatch({ subscribe: msg });
```

Comment:
when these messages were originally queued up for the worker we didn't know if we were going to load the basic worker or the advanced worker and the code in the UI thread forks depending on the option.

Given that we don't know, we queue up the change in the stub worker anyways and if we're loading the advanced worker it'll be processed by it, if we loaded the basic worker it'll basically bounce it back to the UI thread where it can be processed like normal since we now know which worker we're using (and it's already loaded). It's a weird little edge case for these that'll go away if/when we have a single worker type rather than two.